### PR TITLE
Relax pyparsing requirement. 

### PR DESCRIPTION
**Pull request by vEpiphyte**
All oletools unit tests pass with pyparsing 3.0.9 installed.
This makes it easier to use oletools as a library.

### CODE REVIEW

**Comment by decalage2** [comment]
If I remember well, the requirement to keep pyparsing 2.x is due to ViperMonkey, which does not work with pyparsing 3 (and requires oletools).
But if you only use oletools without ViperMonkey indeed it may work.

**Comment by vEpiphyte** [comment]
I see that specific constraint for ViperMonkey here https://github.com/decalage2/ViperMonkey/blob/master/requirements.txt#L6

The vipermonkey tests don't currently run python 3 so I haven't explored if any tests fail on pyparsing 3.x yet.

**Comment by CRogos** [comment]
We are also getting version conflicts in odoo. We need a version that supports pyparsing>3
Couldn't we restrict ViperMonkey to oletools<=60.1?

**Comment by CRogos** [comment]
I've added a PR https://github.com/decalage2/ViperMonkey/pull/125 to ViperMonkey. @decalage2  I think this project should move on, if it is working with newer pyparsing versions.

**Comment by gbtechnology** [comment]
@vEpiphyte 
@decalage2 
this has been opened over 1 year ago. Is there something blocking this merge?

**Comment by vEpiphyte** [comment]
@gbtechnology I am not a maintainer on this project. I cannot merge it.

**Review by decalage2 — APPROVED** [review]

**Comment by CRogos** [comment]
@decalage2 will you also release a new version so this change gets available via pip?

**Comment by johnlandonwood** [comment]
> @decalage2 will you also release a new version so this change gets available via pip?

Seconding this, would be a huge help.

**Pull request merged**
